commonContent.json

```diff
@@ -81,7 +81,12 @@
         "textAllAgentsBusy": "All our web chat agents are busy helping other people.\n Please try again later or contact us using one of the ways below.",
         "textChatClosed" : "Web chat is now closed. \n Come back Monday to Friday, 8am to 6pm or Saturday, 8am to 2pm. Or contact us using one of the ways below.",
         "textChatAlreadyOpen": "A web chat window is already open.",
-        "textAdditional": "Monday to Friday, 8am to 6pm. \n Saturday, 8am to 2pm"
+        "textAdditional": "Monday to Friday, 8am to 6pm. \n Saturday, 8am to 2pm",
+        "localeUrl": "/assets/locale/en-gb",
```

Comment:
we could probably just use `commonContent[htmlLang]` to set the `locale-url `property on the `web-chat` tag, rather than create a separate entry here